cfme/containers/provider/__init__.py

```diff
@@ -93,6 +93,16 @@ class hawkular(Tab, BeforeFillMixin):  # NOQA
         })
         validate = Button('Validate')
 
+    @View.nested
+    class prometheus(Tab, BeforeFillMixin): # noqa
```

Comment:
please stay consistent with the rest of the module and use `# NOQA`. In-line comments also require 2 spaces before the `#`, see L73 above.

Reply:
Ok, done